x/participationrewards/keeper/distribution.go

```diff
@@ -0,0 +1,74 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ingenuity-build/quicksilver/x/participationrewards/types"
+)
+
+type rewardsAllocation struct {
+	ValidatorSelection sdk.Coins
+	Holdings           sdk.Coins
+	Lockup             sdk.Coins
+}
+
+func (k Keeper) getRewardsAllocations(ctx sdk.Context) rewardsAllocation {
+	var allocation rewardsAllocation
+
+	moduleAddress := k.accountKeeper.GetModuleAddress(types.ModuleName)
+	moduleBalances := k.bankKeeper.GetAllBalances(ctx, moduleAddress)
```

Comment:
Given this is a known entity (will always be QCK), can we query just for the QCK balance of this account - saves the nasty `moduleBalances[0]` references below...

Reply:
This allows for localized ibctesting, where DefaultTestingAppInit sets the base denom to ustake;
I can add another var `baseBalance := moduleBalances[0]` for code clarity if desired...